Assets/Scripts/Observer/UnityObserver.cs

```diff
@@ -3,7 +3,7 @@
 
 public class UnityObserver : NetworkBehaviour {
     //Don't Override Awake
-    protected void Awake( ) {
+    protected virtual void Awake( ) {
```

Comment:
It was necessary before, but then I removed it and did other things so not any more